make stops during signature check

**What happened**:
Make process cannot pass target sign.pub.

**What you expected to happen**:
Ignore missing siganture to sign images and create images as requested.

**How to reproduce it (as minimally and precisely as possible)**:
```
user@localhost:~/gardenlinux$ make clean
rm -rf .build

user@localhost:~/gardenlinux$ make metal
gpg: error reading key: No secret key
No secret key for contact@gardenlinux.io exists, signing disabled
/bin/sh: 1: cannot create .build/sign.pub: Directory nonexistent
make: *** [Makefile:11: sign.pub] Error 2
```

**How to fix**
Directory .build is missing.
Can be added to Makefile to successfully pass target sign.pub.

--- a/Makefile
+++ b/Makefile
@@ -7,6 +7,7 @@ BUILDDIR=.build
 BUILDKEY="contact@gardenlinux.io"
 
 sign.pub:
+	@mkdir -p .build
 	@gpg --list-secret-keys $(BUILDKEY) > /dev/null || echo "No secret key for $(BUILDKEY) exists, signing disabled" 
 	@gpg --export --armor $(BUILDKEY) > $(BUILDDIR)/sign.pub
 	@diff $(BUILDDIR)/sign.pub gardenlinux.pub || echo "Not using the official key"

